infection.json

```diff
@@ -0,0 +1,11 @@
+{
+    "$schema": "https://raw.githubusercontent.com/infection/infection/0.26.0/resources/schema.json",
```

Comment:
Same question, I guess.

Reply:
This is defining the schema for the Json file, you always want to keep that pinned to a static version or the schema could break.